build/Extensions/DepsJsonExtensions.cs

```diff
@@ -9,16 +14,8 @@ internal static class DepsJsonExtensions
 {
     public static string GetFolderRuntimeName(this JsonObject depsJson)
     {
-        var runtimeName = depsJson["runtimeTarget"]["name"].GetValue<string>();
-        var folderRuntimeName = runtimeName switch
-        {
-            ".NETCoreApp,Version=v6.0" => "net6.0",
-            ".NETCoreApp,Version=v7.0" => "net7.0",
-            _ => throw new ArgumentOutOfRangeException(nameof(runtimeName), runtimeName,
-                "This value is not supported. You have probably introduced new .NET version to AutoInstrumentation")
-        };
-
-        return folderRuntimeName;
+        var runtime = depsJson.GetTargetFramework();
+        return NuGetFramework.Parse(runtime).ToString();
     }
 
     public static void CopyNativeDependenciesToStore(this JsonObject depsJson, AbsolutePath file, IReadOnlyList<string> architectureStores)
```

Comment:
Do we need this method? Are you planning to clean this up later?

Reply:
yes, trying to concentrate store changes to separate PR. It may be possible, that it's not needed any more.